sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -648,18 +640,17 @@ def update_manifest_properties(self, *args, **kwargs):
         )
 
     @overload
-    def update_tag_properties(self, repository, tag, properties, **kwargs):
-        # type: (str, str, ArtifactTagProperties, **Any) -> ArtifactTagProperties
+    def update_tag_properties(
+        self, repository: str, tag: str, properties: ArtifactTagProperties, **kwargs: Any
+    ) -> ArtifactTagProperties:
         pass
 
     @overload
-    def update_tag_properties(self, repository, tag, **kwargs):
-        # type: (str, str, **Any) -> ArtifactTagProperties
+    def update_tag_properties(self, repository: str, tag: str, **kwargs: Any) -> ArtifactTagProperties:
```

Comment:
Looks like this overload might also benefit from the keyword-only parameters

Reply:
Updated